module/Makefile.in

```diff
@@ -67,10 +67,8 @@ clean-Linux:
 	@# is defined.  This indicates that kernel modules should be built.
 @CONFIG_KERNEL_TRUE@	$(MAKE) -C @LINUX_OBJ@ M="$$PWD" @KERNEL_MAKE@ clean
 
-	if [ -f @LINUX_SYMBOLS@ ]; then $(RM) @LINUX_SYMBOLS@; fi
-	if [ -f Module.markers ]; then $(RM) Module.markers; fi
-
-	find . -name '*.ur-safe' -type f -print | xargs $(RM)
+	$(RM) @LINUX_SYMBOLS@ Module.markers
```

Comment:
Don't we need to use the `-f` flag here to avoid a error in the case where the file doesn't exist?